apps/L1G/restaurant/restaurant-frontend/src/components/home/HomePageContainer.tsx

```
@@ -0,0 +1,100 @@
+'use client';
+import { Button } from '@/components/ui/button';
+
+import { useState } from 'react';
+import MenuCard from './MenuCard';
+const menuItems = [
+  {
+    id: 1,
+    name: 'Taco',
+    price: '15.6к',
+    image: 'https://img1.wsimg.com/isteam/ip/0d953ba0-4d92-4d01-8e07-da1813873883/Sweetish%20meatballs%20pic.jpg',
```

ESLint

[error] Found a string with entropy 4.11 : "https://img1.wsimg.com/isteam/ip/0d953ba0-4d92-4d01-8e07-da1813873883/Sweetish%20meatballs%20pic.jpg"

(no-secrets/no-secrets)